Aztec/Classes/TextKit/FontProvider.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import UIKit
+
+public class FontProvider {
+
+    private init() {
+
+    }
+
+    static var shared = FontProvider()
```

Comment:
The font fields are public, but the shared instance is internal. We should probably make this one public too?